exams/tasks.py

```diff
@@ -219,7 +223,42 @@ def authorize_exam_runs():
             authorized=False,
             date_first_schedulable__lte=now_in_utc(),
     ):
-        api.bulk_authorize_for_exam_run(exam_run)
+        enrollment_ids_qset = ProgramEnrollment.objects.filter(
+            program=exam_run.course.program).values_list('id', flat=True)
+
+        # create a group of subtasks
+        results = group(
+            authorize_enrollment_for_exam_run.s(enrollment_ids, exam_run)
+            for enrollment_ids in chunks(enrollment_ids_qset)
+        )
 
         exam_run.authorized = True
         exam_run.save()
+
+
+@app.task
```

Comment:
`@app.task(acks_late=True)` so that the task doesn't get acknowledged until it completes